module/sheets/dlc-actor-sheet.mjs

```diff
@@ -116,4 +114,133 @@ export class DLCActorSheet extends ActorSheet {
 
     return context;
   }
+
+  /* -------------------------------------------- */
+
+  /** @inheritdoc */
+  activateListeners(html) {
+    super.activateListeners(html);
+
+    // chip control
+    html.find('.chip-control').click((ev) => this.#onChipControl(ev));
```

Comment:
This wasn't working for me, but it turned out to be throwing an error because I was a player and no GM was connected. I suppose it's more because the buttons are just executing everything as the GM at the moment.

Reply:
Ah yeah, some of it needs to be the GM, because it modifies things players don't have direct access to. Using or converting a chip could just be done as the player. Drawing a chip needs the GM.